assets/js/components/data-table.js

```diff
@@ -70,14 +88,14 @@ export const getDataTableFromData = ( data, headers, options ) => {
 				>
 					{ row[ 0 ] === cell && link
 						? <div className="googlesitekit-table__body-item-content">
-							<PrimaryLink
-								className="googlesitekit-table__body-item-link"
-								href={ useAdminURLs ? getSiteKitAdminURL( 'googlesitekit-dashboard', { permaLink } ) : link }
-								external={ ! useAdminURLs }
-								inherit
+							<PrimaryAdminLink
+								permaLink={ permaLink }
+								useAdminURLs={ useAdminURLs }
+								PrimaryLink={ PrimaryLink }
```

Comment:
I was curious about this component being passed around; it turns out it isn't actually used anywhere this function is called:

* https://github.com/google/site-kit-wp/blob/dc6ccb44c39e12a968c55942273b3dc77d427562/assets/js/modules/search-console/components/dashboard/LegacySearchConsoleDashboardWidgetKeywordTable.js#L103-L109
* https://github.com/google/site-kit-wp/blob/dc6ccb44c39e12a968c55942273b3dc77d427562/assets/js/modules/search-console/components/dashboard/LegacyDashboardWidgetPopularKeywordsTable.js#L102-L108
* https://github.com/google/site-kit-wp/blob/dc6ccb44c39e12a968c55942273b3dc77d427562/assets/js/modules/analytics/components/dashboard/LegacyAnalyticsDashboardWidgetTopAcquisitionSources.js#L108-L113
* https://github.com/google/site-kit-wp/blob/dc6ccb44c39e12a968c55942273b3dc77d427562/assets/js/modules/analytics/components/dashboard/LegacyAnalyticsDashboardWidgetPopularPagesTable.js#L91-L99

So let's remove it entirely and just use the `Link` component directly rather than passing this around. 🙂